### PR TITLE
application.html updated with social buttons in center

### PR DESCRIPTION
**Pull request by dklounge**
### Notes on footer social buttons
- Used existing awesomefont
- No change to css by adding text-align to aside element directly in `application.html.haml`
- 3 social buttons added, only twitter with url

### CODE REVIEW

File: app/views/layouts/application.html.haml [hunk 1]
```diff
@@ -24,10 +24,18 @@
...
+        %a{:href => "https://twitter.com/forkingrecipes", :target => "_blank"}
+        %i.fa.fa-twitter-square.fa-3x
+        %a{:href => "#", :target => "_blank"}
+          %i.fa.fa-facebook-square.fa-3x
+        %a{:href => "#", :target => "_blank"}          
+          %i.fa.fa-google-plus-square.fa-3x
+
         .copyright
           Copyright &copy; 2014, 
           %a{:href => "https://twitter.com/forkingrecipes", :class=>"forking-recipes"}
```
Comment (dpick) [inline app/views/layouts/application.html.haml:41]: Should we remove this link now?

Comment (dklounge) [inline app/views/layouts/application.html.haml:41]: Sorry. Which link? Twitter?  Let me know what you expect and I can make changes.

Comment (dpick) [inline app/views/layouts/application.html.haml:41]: Yeah the duplicated twitter link with the odd 'forking-recipes' class.

Comment (dklounge) [inline app/views/layouts/application.html.haml:41]: Ah gotcha - well, I'm no designer. I don't see an issue with it, but if certainly it is redundant.  We could simply keep the text name.

**Comment by dpick** [comment]
Alrighty, I'll merge this in.

**Pull request merged**